src/store/modules/gameList.js

```diff
@@ -8,51 +8,75 @@ class GameSummary {
     this.numPlayers = Object.prototype.hasOwnProperty.call(obj, 'players') ? obj.players.length : 0;
     this.status = Object.prototype.hasOwnProperty.call(obj, 'status') ? obj.status : false;
     this.isRanked = Object.prototype.hasOwnProperty.call(obj, 'isRanked') ? obj.isRanked : false;
+    this.isOver = false;
   }
 }
 export default {
   state: {
-    games: [],
+    openGames: [],
+    spectateGames: [],
```

Comment:
Would it make more sense to keep games, and then have two computed getters that are filtered versions of the state for caching purposes (`spectatableGames` and `openGames`)?

Reply:
Yeah I considered the same and I'm still on the fence. I think a part of the weirdness is that the queries for these two lists of games are different so the backend already has them in different arrays and it seems natural to just keep them separate at that point rather than combining them again in order to separate them via getters.

That said maybe we could rework the query to return both types of games in a single query and then it might make sense to separate them via client side getters

Reply:
Sounds like a follow up to me :)

Reply:
And possibly one that rolls into the change regarding `status` and `result` which I think are confusing and could possibly be combined